.github/workflows/nightly.yml

```diff
@@ -0,0 +1,62 @@
+name: Nightly tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 10 * * *"
```

Comment:
Most days the daily lands between 7:30-9:30 - see dates here: https://nodejs.org/download/nightly/